app/src/views/DrefDetail/index.tsx

```diff
@@ -1,54 +1,56 @@
-import { Container } from '@ifrc-go/ui';
+import {
+    Container,
+    ListView,
+} from '@ifrc-go/ui';
 import { useTranslation } from '@ifrc-go/ui/hooks';
 
 import SurgeContentContainer from '#components/domain/SurgeContentContainer';
 import Link from '#components/Link';
 
 import i18n from './i18n.json';
-import styles from './styles.module.css';
 
 /** @knipignore */
 // eslint-disable-next-line import/prefer-default-export
 export function Component() {
     const strings = useTranslation(i18n);
 
     return (
-        <Container
-            className={styles.drefDetail}
-            contentViewType="vertical"
-            childrenContainerClassName={styles.content}
-        >
-            <SurgeContentContainer
-                heading={strings.drefIntroHeading}
+        <Container spacing="md">
```

Comment:
`spacing="md"` is the default behaviour
```suggestion
        <Container>
```